README.md

```diff
@@ -18,99 +18,112 @@ On an on-going basis there will be:
-- /asset_thumbnails/ is for asset thumbnails (.jpg)
-- /asset_transcripts/ is for asset transcript files (.xml)
+Completing that command may take some time because it's ingesting to and restarting both servers' indexes.
 
-Copy Directory of Files to S3:
+## Verify Successful Ingest
+To verify ingest completed successfully you can view the most recent ingest log files on both the demo and live servers.
 ```
-aws s3 cp /local/folder/of/stuff s3://openvault.wgbh.org/bucket-name --recursive
+$ ssh -i ~/.ssh/OPENVAULT_SSH_KEY_FILE ec2-user@12.345.678.910
 ```
 
-Double Check Files Were Uploaded:
+To get the actual ip addresses, check the [Server Resources](https://wiki.wgbh.org/display/MLA/Server+Resources) wiki page.
 ```
-aws s3 ls s3://openvault.wgbh.org/bucket-name --recursive >> /Users/logs/s3_proxies.csv
+$ cd /var/www/openvault/current/log
+$ ls -l
+$ less ingest-same-mount-files.log
 ```
 
-### What Does Digitized Mean?
-Records on Open vault that are "digitized" means they have a media file available to be viewed online, either on Open Vault, Boston Local TV News Digital Library, or the American Archive of Public Broadcasting websites.
+View the most recent log file.  At the end of the log there should be a % complete number.  If it's `100%` then the ingest was successful.
 
-How a the website knows an asset record is digitized or not is based on either of these two PBCore elements in the metadata.
-```
-<pbcoreAnnotation annotationType="Digitized">YES</pbcoreAnnotation>
-```
-This is used if the record has an asset proxy file that is hosted on the Open Vault S3 bucket and available to view on Open Vault and follows the naming convention.
+If the ingest was not 100% then you need to review the log file and determine why the failing records are failing, correct the data, then re-import those records.
+
+Do the same log, validation check for the demo server, using the other ip address on the [Server Resources](https://wiki.wgbh.org/display/MLA/Server+Resources) wiki page.
+
+There may be instances where the ingest is successful on the live site but not the demo.  This could be because code changes that are currently deployed to the live site that would allow xml to be valid have not yet been deployed to the now demo site.  In those cases, follow the Deploy Code to Demo Server instructions and re-ingest the same xml.
+
+Once you've verified the ingest was 100% successful, you should spot check the records themselves on the live and sites.
 
-To classify a record as digitized that is available on either Boston Local or AAPB, use this element.
-```
-<pbcoreAnnotation annotationType="Outside URL">URL_TO_OUTSIDE_RECORD_PAGE</pbcoreAnnotation>
-```
-It is important to note that if there is an outside url linked for Open Vault asset records, the media files must be available to be viewed on Boston Local or the AAPB online reading room.  Outside urls should not be used for meatadata only records.
```

Comment:
@mccalluc I'm capturing all that in the internal wiki because it's more relevant to how we're creating the records and media rather than the functions of the application itself unless you think it belongs here.